store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/impl/RealStore.kt

```diff
@@ -87,22 +90,42 @@ internal class RealStore<Key : Any, Network : Any, Output : Any, Local : Any>(
         scope = scope,
         realFetcher = fetcher,
         sourceOfTruth = this.sourceOfTruth,
-        converter = converter
+        converter = converter,
+        processor = processor
     )
 
     @Suppress("UNCHECKED_CAST")
-    override fun stream(request: StoreReadRequest<Key>): Flow<StoreReadResponse<Output>> =
-        flow {
-            val cachedToEmit = if (request.shouldSkipCache(CacheType.MEMORY)) {
-                null
-            } else {
+    private suspend fun cachedToEmitOrNull(request: StoreReadRequest<Key>): Output? {
```

Comment:
maybe makes more sense to override stream in the RealStatefulStore and keep the logic there?